conda-recipe/meta.yaml

```diff
@@ -45,7 +45,7 @@ test:
   requires:
     - pytest
     - pytest-timeout
-    - matplotlib
+    - matplotlib <3.9  # avoid installing qt6
```

Comment:
I don't this is appropriate as a CI test dependency, surely it still runs without it?
```suggestion
```

Reply:
There's this one pesky test... https://github.com/pcdshub/pcdsdevices/blob/75d68e72a1d4bcdedef6295c1ba4adec7474f25e/pcdsdevices/tests/test_interface.py#L65-L68

I can try removing it but I think this loads a backend

Reply:
oh sorry, I didn't realize this- but in this case, we need to specify pyqt5 in the test requires if it doesn't work, rather than pinning back matplotlib

Reply:
uh it works.  I don't understand but I'm not going to fight the machine.  Maybe because lightpath pulls in qt dependencies or something i dunno

Reply:
it's worth noting that we need to render in offline/offscreen mode anyway for a CI build- it's not clear that qt would technically be needed with some configuration effort